Dockerfile

```diff
@@ -0,0 +1,54 @@
+ARG HYRAX_IMAGE_VERSION=3.1.0
+FROM ghcr.io/samvera/hyku/hyku-base:$HYRAX_IMAGE_VERSION as ams-base
+
+USER root
+
+RUN apk --no-cache upgrade && \
+  apk --no-cache add \
+    bash \
+    cmake \
+    curl \
+    curl-dev \
+    exiftool \
+    ffmpeg \
+    less \
+    libcurl \
+    libxml2-dev \
+    mariadb-dev \
+    mediainfo \
+    nodejs \
+    openjdk11-jre \
+    openssh \
+    perl \
+    pkgconfig \
+    rsync \
+    vim \
+    yarn && \
+  # curl https://sh.rustup.rs -sSf | sh -s -- -y && \
+  # source "$HOME/.cargo/env" && \
+  # cargo install rbspy && \
+  echo "******** Packages Installed *********"
+
+USER app
+
+RUN mkdir -p /app/fits && \
+    cd /app/fits && \
+    wget https://github.com/harvard-lts/fits/releases/download/1.5.0/fits-1.5.0.zip -O fits.zip && \
+    unzip fits.zip && \
+    rm fits.zip && \
+    chmod a+x /app/fits/fits.sh
+ENV PATH="${PATH}:/app/fits"
```

Comment:
as long as no one every uploads any files, then no. if we have it in the image and someone does upload an image at least it will work. I left it mostly becauseI thought it was installed on your cloud-formation generated machines.

Reply:
👍

Reply:
this one failed when we started. Once we get merged it is probably a good time to pull the skip off and see what's going on.